modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/TranslationEntryManagementToolbarDisplayContext.java

```diff
@@ -78,7 +78,7 @@ public TranslationEntryManagementToolbarDisplayContext(
 					"/translation/delete_translation_entry");
 
 				dropdownItem.putData(
-					"deleteranslationEntriesURL", portletURL.toString());
+					"delete-translation-entries-url", portletURL.toString());
```

Comment:
Currently, for what I've seen in the code, all data keys follow the camelCase convention. Not sure why we need to change it here specifically.

Reply:
Other than this (and the 'delete-url' one), I'm ok with this PR.

Reply:
@adolfopa Why here? I don't want to make the problem bigger but you are right it's spread around the portal :( 

I have two main reasons for these change

- Follow the HTML specification looks like a good idea https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes
- These props end renderer in the HTML, react is complaining about it.
  ![image](https://user-images.githubusercontent.com/67901/111312289-6cc81880-865f-11eb-8b14-54de5f008fb6.png)

Reply:
It was a wrong convention, nothing to worry about (in theory) while using old components, but now React, as @boton mention, complains, becauase standard attributes convention should be followed. From frontend-infra we've been changing all these usages during the migrations that required the implementation of propsTransformers. So, yes, we should go with `data-attribute`.

cc @julien

Reply:
@carloslancha what do you think? 
Maybe it's better to postpone this data convention if exists any plans to fix this globally. 
Anyway, I prefer to do it in the right way now.

Reply:
@adolfopa, I agree with @boton and @carloslancha.

I know we've been using `data-*` attributes in camelCase and "no one" was complaining but now that we use React we do get warnings: it will work, but we're happier without warnings.

You can find more information [here](https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes) and [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*); the **TL;DR** version is:

> The name must not start with xml (case-insensitive).
> The name must not contain any colon characters (:).
> The name must not contain any capital letters.

Reply:
Nothing to add then :-)

Let's move this forward to Brian :+1: